src/en/collectedcurios/build.gradle

```
@@ -0,0 +1,12 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlinx-serialization'
+
+ext {
+    extName = 'Collectedcurios by Jolly Jack'
```

Can name it like the source itself.

```suggestion
    extName = 'Collected Curios'
```